pkg/common/rotationutil/rotationutil.go

```diff
@@ -5,13 +5,19 @@ import (
 	"math/rand"
 	"time"
 
+	"github.com/sirupsen/logrus"
+
 	"github.com/spiffe/spire/pkg/agent/client"
 )
 
+const (
+	gracePeriodThreshold = 12 * time.Hour
```

Comment:
QQ: may we have the same `gracePeriodThreshold` for x509 and jwt? it looks to me like jwt ones must live less time,
and I'm not sure if we even want to have this for those SVIDs... and i case we have it 12hs looks like a lot of time for a minimun 
what do you think?

Reply:
I think that the consideration about JWT-SVIDs is a valid point.
The default JWT-SVID TTL is 5m, and users in general will want a much shorter JWT-SVID TTL than the X509-SVID TTL.
@hiyosi What's the JWT-SVID TTL in your use case? Or was it based mainly on requirements for X509-SVID?
Knowing more about how JWT-SVIDs are issued and consumed in your use case would help to have a better idea about what's the best approach for this in regards to JWT-SVIDs.

Reply:
Originally, I proposed a feature that would only affect X509.
Then, based on the discussion in the proposal, I  implemented a configurable that affects both X509 and JWT. 
But it may not be the final conclusion.

https://github.com/spiffe/spire/issues/4115#issuecomment-1531937467

Reply:
@hiyosi We discussed this again and we feel that it would be kind of weird if this configurable affects JWT-SVIDs also. We think that JWT-SVIDs should be excluded, and have that documented properly. There is no need to make a change in the configuration name, but the implementation should be updated to not affect JWT-SVIDs. The documentation should then be clear that this affects agent SVIDs and workload X509-SVIDs, but not JWT-SVIDs.

Reply:
Thank you for reply. 

> We think that JWT-SVIDs should be excluded, 

I understand. No problem for me either.